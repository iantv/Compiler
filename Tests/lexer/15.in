(a.y > b.y) ? a.y  : (a.y + b.y, d.y = a.y);