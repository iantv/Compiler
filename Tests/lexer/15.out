1	1	(		(
1	2	a		identifier
1	3	.		.
1	4	y		identifier
1	6	>		>
1	8	b		identifier
1	9	.		.
1	10	y		identifier
1	11	)		)
1	13	?		?
1	15	a		identifier
1	16	.		.
1	17	y		identifier
1	20	:		:
1	22	(		(
1	23	a		identifier
1	24	.		.
1	25	y		identifier
1	27	+		+
1	29	b		identifier
1	30	.		.
1	31	y		identifier
1	32	,		,
1	34	d		identifier
1	35	.		.
1	36	y		identifier
1	38	=		=
1	40	a		identifier
1	41	.		.
1	42	y		identifier
1	43	)		)
1	44	;		;